cdap-distributions/src/COPYRIGHT/ngInfiniteScroll@1.2.1/COPYRIGHT

```diff
@@ -0,0 +1 @@
+The MIT License (MIT)
```

Comment:
Is this intended to be a single-line file?

Reply:
Addressed